### ISSUE TITLE
Is this in Julia now?

### ISSUE THREAD

**Issue by haakon-e** [BioTurboNick/AbbreviatedStackTraces.jl#70]
Could you confirm if the code / proposal in this package essentially exists in Julia now?

i.e. starting from v1.10, it should no longer be necessary to use this? If so, could the readme me updated to reflect this progress?

**Comment by BioTurboNick** [comment]
No, it does not exist in Julia now. There have been a handful of improvements to display of stack traces, mostly in types and removing a small subset of kinds of frames.
